packages/web-react/src/components/Button/stories/ButtonLoading.tsx

```diff
@@ -0,0 +1,81 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
+import React from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore: No declaration file
+import icons from '@lmc-eu/spirit-icons/dist/icons';
+import { IconsProvider } from '../../../context';
+import { Icon } from '../../Icon';
+import Button from '../Button';
+
+// @see: https://github.com/storybookjs/storybook/issues/8104#issuecomment-932310244
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const Story = (props: unknown) => (
+  <IconsProvider value={icons}>
```

Comment:
No need to embed `IconsProvider` anymore. It is set globally for the entire storybook. I still have to go through and remove the others. -> `IconGlobalDecorator.js`